src/SprykerEco/Client/Adyen/AdyenClientInterface.php

```diff
@@ -51,4 +51,17 @@ public function handleOnlineTransferResponseFromAdyen(AdyenRedirectResponseTrans
      * @return \Generated\Shared\Transfer\AdyenRedirectResponseTransfer
      */
     public function handleCreditCard3dResponseFromAdyen(AdyenRedirectResponseTransfer $redirectResponseTransfer): AdyenRedirectResponseTransfer;
+
+    /**
+     * Specification:
+     * - Handle response from Adyen after redirect customer back to the shop after Klarna Invoice has been passed.
+     * - Save response, update status etc.
```

Comment:
`Saves`
It's not clear what statuses are going to be updated. Move details would be nice.
And I suppose it's not allowed to put `etc` in specification :)

Reply:
Changed